indico/modules/events/abstracts/controllers/management.py

```diff
@@ -179,6 +179,15 @@ def _process(self):
         return jsonify_form(form)
 
 
+class RHExportBOA(RHManageAbstractsBase):
+    """Export book of abstracts"""
+
+    def _process(self):
+        pdf = AbstractBook(self.event_new)
+        fname = pdf.generate()
```

Comment:
I also think it's cleaner to not assign it to an intermediate variable. But FYI `generate()` actually does return the path to the temporary file containing the PDF